index.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;700&family=Roboto:wght@300;400;500;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="assets/css/style.css" />
+    <style></style>
+    <title>Store</title>
+  </head>
+  <body>
+    <header>
+      <div class="container">
+        <div class="header__top">
+          <div class="header__logo">
+            <a href="/">Logo</a>
+          </div>
+          <div class="header__controls">
+            <button id="search" class="btn">Search</button>
+            <button class="btn btn--primary">Login</button>
```

Comment:
Adding `type="button"` to button elements is recommended if it's controlled by JS logic. Read up more on MDN ;)